File: app/src/main/java/com/example/pcmall/model/Category.java
package com.example.pcmall.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Data
@Accessors(chain = true)
public class Category {
    private Integer id;//分类编号
    private String cname;//分类名称
    private LocalDateTime createdTime;//创建时间
    private LocalDateTime updateTime;//修改时间
    private Integer isDelete;//是否删除（0正常 1删除）
}

File: core/model/src/main/java/com/example/pcmallcompose/core/model/Category.kt
package com.example.pcmallcompose.core.model

import com.fasterxml.jackson.annotation.JsonFormat
import com.fasterxml.jackson.databind.annotation.JsonDeserialize
import com.fasterxml.jackson.databind.annotation.JsonSerialize
import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer
import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer
import java.time.LocalDateTime

data class Category(
    val id: Int, //分类编号
    val cname: String, //分类名称
    @field:JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    @field:JsonDeserialize(using = LocalDateTimeDeserializer::class)
    @field:JsonSerialize(using = LocalDateTimeSerializer::class)
    val createdTime: LocalDateTime, //创建时间
    @field:JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    @field:JsonDeserialize(using = LocalDateTimeDeserializer::class)
    @field:JsonSerialize(using = LocalDateTimeSerializer::class)
    val updateTime: LocalDateTime? = null,//修改时间
    val isDelete: Int,//是否删除（0正常 1删除）
)